packages/cli/src/openapi/README.md

```diff
@@ -0,0 +1,64 @@
+# appmap-openapi
+
+`@appland/appmap openapi` generates
+[OpenAPI 3](https://openapi.io/specification/) (aka OpenAPI) YAML from
```

Comment:
`https://openapi.io/specification/` is not a live site (s/swagger/opeapi/g artefact?)
The latest spec is at https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md
or https://spec.openapis.org/oas/latest.html (it references the github markdown file as the source of truth)